pulumi/aws/anthos/__main__.py

```diff
@@ -251,26 +274,32 @@ def add_namespace(obj):
 if not anthos_host:
     anthos_host = lb_ingress_hostname
 
-boa_in = k8s.networking.v1beta1.Ingress("boaIngress",
-                                        api_version="networking.k8s.io/v1beta1",
-                                        opts=pulumi.ResourceOptions(depends_on=[ns, boa]),
-                                        kind="Ingress",
-                                        metadata=k8s.meta.v1.ObjectMetaArgs(
-                                            name="bankofanthos",
-                                            namespace=ns
-                                        ),
-                                        spec=k8s.networking.v1beta1.IngressSpecArgs(
-                                            ingress_class_name="nginx",
-                                            rules=[k8s.networking.v1beta1.IngressRuleArgs(
-                                                host=lb_ingress_hostname,
-                                                http=k8s.networking.v1beta1.HTTPIngressRuleValueArgs(
-                                                    paths=[k8s.networking.v1beta1.HTTPIngressPathArgs(
-                                                        path="/",
-                                                        backend=k8s.networking.v1beta1.IngressBackendArgs(
-                                                            service_name="frontend",
-                                                            service_port=80,
-                                                        ),
-                                                    )],
-                                                ),
-                                            )],
-                                        ))
+boaingress = k8s.networking.v1beta1.Ingress("boaingress",
```

Comment:
Now that this block has become more complex, it would be good to intersperse it with comments to tell the story about what you are doing.

Reply:
Lots of additional detail added to that section in comments.